apps/etterlatte-utbetaling/src/test/kotlin/utbetaling/avstemming/AvstemmingDaoIntegrationTest.kt

```diff
@@ -87,22 +91,23 @@ internal class AvstemmingDaoIntegrationTest {
                 til = now.minus(2, ChronoUnit.DAYS)
             ),
             antallOppdrag = 3,
-            avstemmingsdata = ""
+            avstemmingsdata = "",
+            saktype = Saktype.BARNEPENSJON
         )
 
         avstemmingDao.opprettGrensesnittavstemming(grensesnittavstemming1)
         avstemmingDao.opprettGrensesnittavstemming(grensesnittavstemming3)
         avstemmingDao.opprettGrensesnittavstemming(grensesnittavstemming2)
 
-        val nyesteAvstemming = avstemmingDao.hentSisteGrensesnittavstemming()
+        val nyesteAvstemming = avstemmingDao.hentSisteGrensesnittavstemming(Saktype.BARNEPENSJON)
```

Comment:
Hvorfor ikke sjekke equals på hele objektet under? Mye sikrere og tydeligere for den som leser testen hvilket objekt som er forventet.

Reply:
Det var en god idé, implementert nå.